pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -13,6 +13,6 @@
     .row
       .col-md-6
         =view :signup_form
-      .col-md-6
-        -if showAnnouncements && hoc_mode != "pre-hoc"
+      -if showAnnouncements && @country == "us" && hoc_mode != "pre-hoc"
```

Comment:
Also, do we need to think through the hoc_modes like "post-hoc"?  Or is anything but "pre-hoc" appropriate?

Reply:
I left what was previously there re: `pre-hoc`, but I did check all of the modes and it works as expected: announcements will only show during `soon-hoc` since `actual-hoc` and `post-hoc` don't show the form, and it's just the form during `pre-hoc`.

Reply:
Ah nice.